File: CS/DiagnosticTool/LinuxCommands.cs
namespace DiagnosticTool.LinuxTools {
    public class LinuxCommands {
        public static string StartSession = $"lttng create dx-session -o./dx-diagnostic-trace \n lttng enable-channel --userspace DotNetCoreChannel \n lttng add-context --userspace --channel=DotNetCoreChannel --type=vtid \n lttng enable-event --userspace --tracepoint DotNETRuntime:EventSource --channel=DotNetCoreChannel --filter \"EventSourceName==\'DX-Telemetry\'\" \n lttng start";

        public static string StopSession(string path) => $"lttng stop \n lttng view >> {path} \n lttng destroy dx-session \n rm -r dx-diagnostic-trace";
    }
}

File: VB/DiagnosticTool/LinuxCommands.vb
Namespace DiagnosticTool.LinuxTools

    Public Class LinuxCommands

        Public Shared StartSession As String = $"lttng create dx-session -o./dx-diagnostic-trace 
 lttng enable-channel --userspace DotNetCoreChannel 
 lttng add-context --userspace --channel=DotNetCoreChannel --type=vtid 
 lttng enable-event --userspace --tracepoint DotNETRuntime:EventSource --channel=DotNetCoreChannel --filter ""EventSourceName=='DX-Telemetry'"" 
 lttng start"

        Public Shared Function StopSession(ByVal path As String) As String
            Return $"lttng stop 
 lttng view >> {path} 
 lttng destroy dx-session 
 rm -r dx-diagnostic-trace"
        End Function
    End Class
End Namespace
